Objects/exceptions.c

```diff
@@ -78,6 +78,33 @@ BaseException_init(PyBaseExceptionObject *self, PyObject *args, PyObject *kwds)
     return 0;
 }
 
+
+static PyObject *
+BaseException_vectorcall(PyObject *type_obj, PyObject * const*args,
+                         size_t nargsf, PyObject *kwnames)
+{
+    PyTypeObject *type = _PyType_CAST(type_obj);
+    PyBaseExceptionObject *self;
+
+    self = (PyBaseExceptionObject *)type->tp_alloc(type, 0);
+    if (!self)
+        return NULL;
+    /* the dict is created on the fly in PyObject_GenericSetAttr */
+    self->dict = NULL;
+    self->notes = NULL;
+    self->traceback = self->cause = self->context = NULL;
```

Comment:
Nit: I prefer spelling these out instead of cluttering them together on one line.

```suggestion
    self->traceback = NULL;
    self->context = NULL;
    self->cause = NULL;
```

Reply:
````````